client/blocks/dismissable-card/README.md

```diff
@@ -0,0 +1,67 @@
+Dismissable Card
+=========
+This is a card component that can be dismissed for a single page load or be hidden
+via user preference.
+
+#### How to use:
+
+```js
+import DismissableCard from 'blocks/dismissable-card';
+import QueryPreferences from 'components/data/query-preferences';
+
+render: function() {
+  return (
+    <div className="your-stuff">
+      <QueryPreferences />
```

Comment:
Relying on the presence of QueryPreferences up the tree seems like a strange coupling to do. Though I think the QueryPreferences should be placed within the dismissable-card component if possible.